core/src/replay_stage.rs

```diff
@@ -2084,7 +2084,7 @@ impl ReplayStage {
         bank_forks: &Arc<RwLock<BankForks>>,
         poh_recorder: &Arc<RwLock<PohRecorder>>,
         leader_schedule_cache: &Arc<LeaderScheduleCache>,
-        rpc_subscriptions: &Arc<RpcSubscriptions>,
+        rpc_subscriptions: &Option<Arc<RpcSubscriptions>>,
```

Comment:
On the flipside, probably not worth changing here for consistency with every other arg :upside_down_face: